CHANGELOG.md

```diff
@@ -12,7 +16,8 @@
 
 ### 2.4.4
 
-- Add support for non ISO-8859-1 characters in request headers to the Cortex API. This fixes authentication for users with these characters in their name.
+- Add support for non ISO-8859-1 characters in request headers to the Cortex API. This fixes authentication for users
```

Comment:
Hmm could we revert the formatting changes to these other entries please?

Reply:
had hard time struggling with webstorm on this :)